src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs

```diff
@@ -67,16 +67,34 @@
                    (str "Selected Method Configuration Namespace : " (:selected-mc-namespace @state))
                    [:hr]
                    "Destination Name : "
-                   (style/create-text-field {:style {:ref "destinationName"}})
+                   (style/create-text-field {:id "dest_name_id" :style {:ref "destinationName"}})
                    [:br]
                    "Destination Namespace : "
-                   (style/create-text-field {:style {:ref "destinationNamespace"}})
+                   (style/create-text-field {:id "dest_namespace_id" :style {:ref "destinationNamespace"}})
                    [:br]
+                   [comps/Button  {:title_text "Pre-populate names"
+                                   :icon :gear
+                                   :onClick (fn []
+                                              (let [
+                                                 dest_name_elem  (.getElementById js/document (name "dest_name_id"))
+                                                 method_config_prepopname ((:selected-conf @state) "name")
+                                                 dest_namespace_elem (.getElementById js/document (name "dest_namespace_id"))
+                                                 method_config_ns_prepopname ((:selected-conf  @state) "namespace" )
+                                                 ]
+                                                (set! (.-value dest_name_elem) method_config_prepopname)
+                                                (set! (.-value  dest_namespace_elem)  method_config_ns_prepopname)))}]
                    [comps/Button
                     {:icon :plus
-                     :onClick (fn [] (swap! state assoc :show-import-mc-modal? false))}]]]
-        :show-when (:show-import-mc-modal? @state)
-        :dismiss-self (fn [] (swap! state assoc :show-import-mc-modal? false))}]
+                     :onClick (fn []
+                                (utils/rlog "this is where an AJAX call must be made!")
+                                (swap! state assoc :show-import-mc-modal? false)
+                                )}]
+                   [comps/Button
```

Comment:
I don't like this X button like this.  Either have buttons along the bottom named "Okay" and "Cancel" (for example, the Create New Workspace dialog), or have an okay button along the bottom and an "X" button in the top corner.